pagerduty/resource_pagerduty_event_orchestration_path_router_test.go

```diff
@@ -34,12 +35,32 @@ func TestAccPagerDutyEventOrchestrationPathRouter_Basic(t *testing.T) {
 		Regex:    "ID:(.*)",
 		Source:   "event.custom_details.pd_service_id",
 	}
+	invalidDynamicRouteToPlacementMessage := "Invalid Dynamic Routing rule configuration:\n- A Router can have at most one Dynamic Routing rule; Rules with the dynamic_route_to action found at indexes: 1, 2\n- The Dynamic Routing rule must be the first rule in a Router"
+	invalidDynamicRouteToConfigMessage := "Invalid Dynamic Routing rule configuration:\n- Dynamic Routing rules cannot have conditions\n- Dynamic Routing rules cannot have the `route_to` action"
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckPagerDutyEventOrchestrationRouterDestroy,
 		Steps: []resource.TestStep{
+			// Invalid Dynamic Routing rule config for a new resource: multiple Dynamic Routing rules, Dynamic Routing rule not the first rule in the Router:
```

Comment:
**Annotation:** I wanted to make sure that validation works for both new and existing resources so I added the invalid config tests in the beginning, when the resource doesn't exist, and after a dynamic rule was created on the resource.